### PR TITLE
Rabbit mq

### PR DESCRIPTION
**Pull request by Ahmed-anwar**

### CODE REVIEW

**Comment by HZahran** [comment]
Reviewed

**Comment by Ahmed-anwar** [comment]
@HZahran issues are resolved.

**Comment by Ahmed-anwar** [comment]
@HZahran new issues are also resolved.

File: src/main/java/com/linkedin/replica/connections/commands/impl/AcceptFriendCommand.java [hunk 1]
```diff
@@ -15,6 +15,8 @@ public AcceptFriendCommand(HashMap<String, Object> args){
 	}
 	@Override
 	public Object execute() throws SQLException {
+		validateArgs(new String[]{"userID1"});
```
Comment (HZahran) [inline src/main/java/com/linkedin/replica/connections/commands/impl/AcceptFriendCommand.java]: can be validateArgs(new String[]{"userID1","userID2"});

**Comment by HZahran** [comment]
Reviewed last changes (Y)

**Pull request merged**